csrc/custom/paged_attention/attention_ll4mi.cu

```diff
@@ -709,17 +769,17 @@ __launch_bounds__(NUM_THREADS) void paged_attention_ll4mi_reduce_kernel(
   float acc = 0.0f;
   #pragma unroll
   for (int j = 0; j < JCHUNK; j++) {
-    acc += tmps[j] * shared_exp_sums[j];
+    acc += to_float<scalar_t>(tmps[j]) * shared_exp_sums[j];
```

Comment:
Is here the case that compiler won't convert `bfloat16` to `float` automatically?

Reply:
IIRC 6.1 compiler did not do the conversion automatically, but 6.2 compiler does auto conversion.